CHANGELOG/CHANGELOG-1.9.3.md

```diff
@@ -0,0 +1,75 @@
+## SUSE Storage v1.9.3 Release Notes
+
+SUSE Storage 1.9.3 introduces several improvements and bug fixes that are intended to improve system quality, resilience, stability and security.
+
+The SUSE Storage team appreciates your contributions and expects to receive feedback regarding this release.
+
+> [!NOTE]
+> For more information about release-related terminology, see [Releases](https://github.com/rancher/suse-storage/releases).
+
+## Installation
+
+>  [!IMPORTANT]
+**Ensure that your cluster is running Kubernetes v1.25 or later before installing SUSE Storage v1.9.3.**
+
+You can install SUSE Storage using a variety of tools, including Rancher, Kubectl, and Helm. For more information about installation methods and requirements, see [Installation](https://documentation.suse.com/cloudnative/storage/1.9/en/installation-setup/installation/installation.html) in the SUSE Storage documentation.
+
+## Upgrade
+
+>  [!IMPORTANT]
+**Ensure that your cluster is running Kubernetes v1.25 or later before upgrading from SUSE Storage v1.9.x (< v1.9.3) to v1.9.3.**
```

Comment:
We should mention upgrading from 1.8 community is also allowed or concerns? Point users to refer to the Migration section.